public/js/msg.js

```diff
@@ -104,7 +110,7 @@ window.HLRDESK.init.messages = function() {
 
   //newTask
   socket.on('write task', function(task){
-    $('#task_sort').append($("<div class='projects' id='"+task.task_id+"'></div>").text(task.text));
+    if(!taskName)$('#task_sort').append($("<div class='projects' id='"+task.task_id+"'></div>").text(task.text)).one();
```

Comment:
What is `.one()` doing? From the documentation it looks like it's for adding events, but none are passed here.